recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: cd2aac2b566c7e740d34aabb4737864c74ba33c29aad7101fbf1bab2931c02dc
 
 build:
-  number: 0
+  number: 1
   skip: true  # [py<39]
```

Comment:
I assume that we can do like scipy:

https://github.com/conda-forge/scipy-feedstock/blob/main/recipe/meta.yaml#L43C3-L44C40

Reply:
Thanks, I opened https://github.com/conda-forge/scikit-learn-feedstock/pull/266 to do this